protocols/delegate-manager/test/DelegateManager.js

```diff
@@ -0,0 +1,141 @@
+const ERC20 = artifacts.require('FungibleToken')
+const Swap = artifacts.require('Swap')
+const Indexer = artifacts.require('Indexer')
+const Delegate = artifacts.require('Delegate')
+const DelegateFactory = artifacts.require('DelegateFactory')
+const DelegateManager = artifacts.require('DelegateManager')
+const { equal, passes } = require('@airswap/test-utils').assert
+const { padAddressToLocator } = require('@airswap/test-utils').padding
+
+contract('DelegateManager Integration Tests', async accounts => {
+  const STARTING_BALANCE = 700
+  const INTENT_AMOUNT = 250
+  let owner = accounts[0]
+  let notOwner = accounts[2]
+  let tradeWallet = accounts[1]
+  let DAI_TOKEN
+  let WETH_TOKEN
+  let stakeToken
+  let swap
+  let indexer
+  let delegateFactory
+  let delegateManager
+  let delegateAddress
+
+  async function setupTokenAmounts() {
+    await stakeToken.mint(owner, STARTING_BALANCE)
+    await DAI_TOKEN.mint(owner, STARTING_BALANCE)
+    await WETH_TOKEN.mint(owner, STARTING_BALANCE)
+
+    await stakeToken.mint(notOwner, STARTING_BALANCE)
+    await DAI_TOKEN.mint(notOwner, STARTING_BALANCE)
+    await WETH_TOKEN.mint(notOwner, STARTING_BALANCE)
+  }
+
+  before(async () => {
+    DAI_TOKEN = await ERC20.new()
+    WETH_TOKEN = await ERC20.new()
+    stakeToken = await ERC20.new()
+    await setupTokenAmounts()
+
+    swap = await Swap.new()
+
+    delegateFactory = await DelegateFactory.new(swap.address)
+    delegateManager = await DelegateManager.new(
+      delegateFactory.address,
+      tradeWallet
+    )
+
+    indexer = await Indexer.new(stakeToken.address, delegateFactory.address)
+    await indexer.createIndex(DAI_TOKEN.address, WETH_TOKEN.address)
+  })
+
+  describe('Test initial values', async () => {
+    it('Test factory address', async () => {
+      let val = await delegateManager.factory.call()
+      equal(val, delegateFactory.address, 'factory was not properly set')
+    })
+
+    it('Test delegate address', async () => {
+      delegateAddress = await delegateManager.delegate.call()
+      //TODO: ensure it's not empty
```

Comment:
should be able to test this using
```
assert.notEqual(EMPTY_ADDRESS, delegateAddress)
```
this would mean adding `notEqual` to the assert utils. Alternatively you could just do
```
assert.ok(EMPTY_ADDRESS != delegateAddress)
```
`ok` is already in the assert utils - it just checks that an equation equates to true.

Reply:
thanks. forgot to add this test.